force-app/main/default/classes/IBMDiscoveryV1.cls

```diff
@@ -520,9 +520,7 @@ public class IBMDiscoveryV1 extends IBMWatsonService {
     }
     builder.query('version', versionDate);
     final Map<String, Object> contentJson = new Map<String, Object>();
-    if (updateCollectionOptions.name() != null) {
-      contentJson.put('name', updateCollectionOptions.name());
-    }
+    contentJson.put('name', updateCollectionOptions.name());
```

Comment:
is this breaking?

Reply:
It's technically breaking but the name would've always needed to be supplied. We mistakenly didn't consider it to be required in the SDK but the service would've thrown an error without it.

For that reason, I'm considering this change a fix.

Reply:
Approved! I don’t see default service names in here